src/WorkflowContext.ts

```diff
@@ -11,234 +11,80 @@ import { EventEmitter } from "events";
-     * @type {{ (context: WorkFlowContext) }}
-     * @memberOf Plugin
-     */
     init?: { (context: WorkFlowContext) };
-    /**
-     * 
-     * 
-     * @type {{ (file: File, ast?: any) }}
-     * @memberOf Plugin
-     */
     transform?: { (file: File, ast?: any) };
-
-
     transformGroup?: { (file: File) };
-    /**
-     * 
-     * 
-     * @param {WorkFlowContext} context
-     * 
-     * @memberOf Plugin
-     */
     bundleStart?(context: WorkFlowContext);
-    /**
-     * 
-     * 
-     * @param {WorkFlowContext} context
-     * 
-     * @memberOf Plugin
-     */
     bundleEnd?(context: WorkFlowContext);
 }
 
 /**
- * WorkFlowContext
- */
-/**
- * 
- * 
- * @export
- * @class WorkFlowContext
+ * Gets passed to each plugin to track FuseBox configuration
  */
 export class WorkFlowContext {
-
-
     public shim: any;
 
-    /**
-     * 
-     * 
-     * 
-     * @memberOf WorkFlowContext
-     */
     public emmitter = new EventEmitter();
+
     /**
-     * 
-     * 
-     * 
-     * @memberOf WorkFlowContext
+     * The default package name or the package name configured in options
      */
     public defaultPackageName = "default";
 
-    /**
-     * 
-     * 
-     * @type {string[]}
-     * @memberOf WorkFlowContext
-     */
     public ignoreGlobal: string[] = [];
-    /**
-     * 
-     * 
-     * @type {Map<string, ModuleCollection>}
-     * @memberOf WorkFlowContext
-     */
+
     public nodeModules: Map<string, ModuleCollection> = new Map();
-    /**
-     * 
-     * 
-     * @type {Map<string, IPackageInformation>}
-     * @memberOf WorkFlowContext
-     */
+
     public libPaths: Map<string, IPackageInformation> = new Map();
-    /**
-     * 
-     * 
-     * @type {string}
-     * @memberOf WorkFlowContext
-     */
+
     public homeDir: string;
-    /**
-     * 
-     * 
-     * 
-     * @memberOf WorkFlowContext
-     */
-    public printLogs = true;
-    /**
-     * 
-     * 
-     * @type {Plugin[]}
-     * @memberOf WorkFlowContext
-     */
-    public plugins: Plugin[];
 
+    public printLogs = true;
 
+    public plugins: Plugin[];
 
     public fileGroups: Map<string, File>;
-    /**
-     * 
-     * 
-     * 
-     * @memberOf WorkFlowContext
-     */
+
     public useCache = true;
-    /**
-     * 
-     * 
-     * 
-     * @memberOf WorkFlowContext
-     */
+
     public doLog = true;
-    /**
-     * 
-     * 
-     * 
-     * @memberOf WorkFlowContext
-     */
+
     public cache: ModuleCache;
-    /**
-     * 
-     * 
-     * @type {*}
-     * @memberOf WorkFlowContext
-     */
+
     public tsConfig: any;
-    /**
-     * 
-     * 
-     * @type {string}
-     * @memberOf WorkFlowContext
-     */
+
     public customModulesFolder: string;
-    /**
-     * 
-     * 
-     * 
-     * @memberOf WorkFlowContext
-     */
+
     public tsMode = false;
 
-    public loadedTsConfig: any;
-    /**
-     * 
-     * 
-     * @type {string[]}
-     * @memberOf WorkFlowContext
-     */
-    public globals: any;
```

Comment:
the `@type` was inconsistent with docs. Adding a type annotation for fusebox options caught it though :rose: